docs/user/filesystem.md

```diff
@@ -666,19 +667,21 @@ to be set to 0 or off, except dentry-dir-time which defaults to dentry-time
 | dfuse-dentry-time       | How long directory entries are cached                                  |
 | dfuse-dentry-dir-time   | How long dentries are cached, if the entry is itself a directory       |
 | dfuse-ndentry-time      | How long negative dentries are cached                                  |
-| dfuse-data-cache        | Data caching enabled, duration or ("on"/"true"/"off"/"false")          |
+| dfuse-data-cache        | Data caching enabled, duration or ("on"/"true"/"off"/"false"/"otoc")   |
```

Comment:
otoc sound weird and not intuitive on what it is and why is it needed. i don't have a better suggestion though. maybe "on_mmap" or something like that? not requesting a change just an observation.

Reply:
"close to open" consistency is a well known term so I thought "open to close" caching made sense in that regard - and it's clear then what effect this setting will have.